frankly034/Dantown
# Fix login response

#### What does this PR do?
Fixes login error response

#### Description of Task to be completed?
- fix login error object

#### How should this be manually tested?
* clone the repo
* run `npm install`
* start the app with `npm run start:dev`
* visit POST /auth/login
* Enter invalid login credentials
* You should get a `200` status on success or `400` and appropriate message on failure

#### Any background context you want to provide?
* must have `node` installed

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
